app/assets/javascripts/discourse/tests/acceptance/user-preferences-profile-test.js

```diff
@@ -4,6 +4,38 @@ import { cloneJSON } from "discourse/lib/object";
 import userFixtures from "discourse/tests/fixtures/user-fixtures";
 import { acceptance } from "discourse/tests/helpers/qunit-helpers";
 
+acceptance(
```

Comment:
These two tests ensures the checkbox is behind the `allow users to hide profile` site setting.